source/blog/2018-05-30-emberjs-accessibility-and-a11y-tools.md

```diff
@@ -0,0 +1,168 @@
+
+So why is this necessary? If you're looking at a decently accessible website and you hit the tab key, you'll see the focus jump between different elements - things get highlighted. Front end frameworks like Ember sometimes get in the way of the DOM structure that works best for screen reader focus, since as you move between routes, only part of the page changes. The contents of `{{outlet}}` are dynamic. 
+
+The problem is that screen readers rely heavily on what has focus. Imagine if you had to start reading at the very top of the page whenever a route in your app changed, instead of just reading the content that is new, or if you weren't able to tell that some content had changed! ember-a11y corrects for that focus problem so that Assistive Technology navigation using focus works correctly. From the README:
+
+> The current implementation of this addon will immediately apply focus to the most relevant piece of content based on actions users take (clicking buttons, links, etc). This allows screen readers to catch changes and read the right information, thus providing a much better experience for blind users.
+
+## What next?
+
+It's our responsibility as developers to build inclusive tech, so keep learning and building! [Melanie Sumner's](https://github.com/MelSumner) EmberConf talk [Accessibility for All](https://www.youtube.com/watch?v=6ydNf7e5P00) is a fast, concise introduction to accessibility in Ember, and [this article series](http://www.melsumner.com/blog/accessibility/a11y-accessibility-guide-ember-developers/) provides a deeper dive.
+
+There are already many community addons available that solve different issues. While we still have a ways to go, new advances are being made too. For example, there is a [Semantic Test Selectors RFC](https://github.com/emberjs/rfcs/pull/327) being drafted and an [experimental addon](https://github.com/tradegecko/ember-semantic-test-helpers) in development to go with it! These efforts followed [Jamie White's](https://github.com/jgwhite) talk from EmberConf, [Say More](https://www.youtube.com/watch?v=iAyRVPSOpy8).
+
+Almost all of us will have a disability at some point in our lives. Accessible web practices lead to better user experience --- things like improved keyboard navigation, text that is still readable on a dim monitor, and clear pointers on how to interact with a UI. 
+
+Here are some ways you can help improve accessibility in Ember and the JavaScript ecosystem:
+
+- Research and try out other [ember-a11y](https://github.com/ember-a11y) tools
+- Make your own apps accessible
+- Include a11y considerations in your talks and articles
+- Contribute to addons that help solve these problems
+- Guide your coworkers and collaborators through the learning process
+- Avoid language that perpetuates the idea that accessibility is too hard. Focus
+on the tools that make it easier and our responsibilities as developers.
+- Support and comment on RFCs that help make Ember more accessible out of the box
```

Comment:
Having this list of actions is really awesome. Is there a way we can encourage people to share the improvements they make? Like “When you make an a11y improvement to your app, let the world know on Twitter/Medium/etc.”?

We see a lot of wonderful “check out this cool CSS trick” type posts with links on CodePen etc. I think it would be awesome to see people similarly jazzed about implementing delightful accessible UIs.

Reply:
Love it, done